conf/defaults.yaml

```diff
@@ -263,7 +263,7 @@ topology.state.checkpoint.interval.ms: 1000
 # topology priority describing the importance of the topology in decreasing importance starting from 0 (i.e. 0 is the highest priority and the priority importance decreases as the priority number increases).
 # Recommended range of 0-29 but no hard limit set.
 topology.priority: 29
-topology.component.resources.onheap.memory.mb: 128.0
+topology.component.resources.onheap.memory.mb: 256.0
```

Comment:
So system components (i.e. acker and writer) use more memory than 128.0mb. When I turned on cgroups, workers will just system components on them will get consistently killed by cgroups for using more memory than allowed